homework/tests/HomeworkTests.spec.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+
+
+test.describe(version + ': Add', () => {
+    test('Adding 2 and 3 results in 5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
+      await page.locator('#number1Field').type('2');
```

Comment:
why are you using type and not fill function? The latter would also automatically do more checks of the field before entering values. Now when your build 9 tests fail, they do not mention second input as an issue, only the button. Or maybe you could add visibility and similar checks manually?